alpha/lib/model/DeliveryProfileVod.php

```diff
@@ -13,6 +13,22 @@
 	/** -------------------
 	 * Functionality 
 	 * --------------------*/
+
+	protected function getPlayServerUrlPrefix()
+	{
+		return '';
+	}
+
+	protected  function generatePlayServerUrlPrefix()
+	{
+		$prefix = '/usePlayServer/1';
+		if($this->getDynamicAttributes()->getUiConfId())
+			$prefix .= '/uiConfId/'.$this->getDynamicAttributes()->getUiConfId();
+		if($this->getDynamicAttributes()->getSessionID())
+			$prefix .= '/sessionId/'.$this->getDynamicAttributes()->getSessionID();
```

Comment:
We need to consider to generate the session ID here , rather generate it on any request and use it only here. due to performance considerations.